### PR TITLE
tests: Ignore text output of multithreading snapshot test

### PR DESCRIPTION
**Pull request by theduke**
Output is not deterministic.

Prevents CI flakiness.

### CODE REVIEW

**Comment by theduke** [comment]
ps: The test still tests what it's supposed to test due to the way it is written, even without diffing the output.

File: tests/integration/cli/tests/snapshot.rs [hunk 1]
```diff
@@ -89,6 +89,21 @@ pub enum TestResult {
...
 
+impl TestResult {
+    /// Remove stdout and stderr from the result.
+    ///
+    /// Can be used for tests that don't have deterministic output.
+    pub fn ignore_output(&mut self) {
+        match self {
+            TestResult::Success(s) => {
+                s.stdout.clear();
+                s.stderr.clear();
```
Comment (ptitSeb) [inline tests/integration/cli/tests/snapshot.rs:100]: maybe keep stderr?

**Review by ptitSeb — APPROVED** [review]

**Comment by syrusakbary** [comment]
Weird. The python flakiness has come back? Or perhaps is just that the branch was updated. Updating it to see if that solves things

**Review by syrusakbary — APPROVED** [review]

**Comment by syrusakbary** [comment]
Closing as stale

**Pull request closed without merging**